src/dcegm/egm.py

```diff
@@ -343,23 +330,19 @@ def get_child_state_choice_specific_policy(
             (n_choices, n_quad_stochastic * n_grid_wealth).
 
     """
-    next_period_wealth_flat = next_period_wealth.flatten("F")
-
-    next_period_policy_interp = np.empty(
-        (child_node_choice_set.shape[0], next_period_wealth_flat.shape[0])
-    )
+    next_period_policy_interp = np.empty((child_node_choice_set.shape[0], 1))
```

Comment:
The second dimension of the array is just 1 and therefore not needed. Just use a one dimensional array!